.github/workflows/conventional-commit-lint.yaml

```diff
@@ -50,7 +50,6 @@ jobs:
           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
 
       - name: Add NPM token
-        if: steps.cache.outputs.cache-hit != 'true'
         run: npm config set '//npm.intergiro.tech/:_authToken' ${NPM_AUTH_TOKEN}
```

Comment:
Why is  this removed?

Reply:
We really had some errors in repositories, where cache was created, but npm ci needs token

Reply:
Ok but is it connected to "Add cleanup workflow" ? If not I don think we should mix changes in a pull request. I know it is easy to include a small thing but it is not good for tracability, reverting and long term understanding. This change should be a separate pull request.